### DISCUSSION TITLE
Just rebooted after windows update and now my colors are off

### DISCUSSION THREAD

**Discussion by curtiscarlson** [atom/atom#22219] [General]
in php...every $ is now white..while the rest of the variable name is red...its unreadable and hurting my brain, how do I make it stop?

**Comment by curtiscarlson** [comment]
It also now highlights the entire variable when you double click it instead of just the part that does not include the $....which is a downgrade and also not intuitive at all since the $ is now a different color when it used to be the same....

**Reply by ThatXliner** [reply]
Maybe this will help: https://github.com/atom/language-php/issues/423
https://github.com/atom/atom/pull/20524#issuecomment-821261826

**Reply by curtiscarlson** [reply]
seems like a work around and won't solve the double click issue...they should just roll back whatever update was done

**Reply by ThatXliner** [reply]
Well https://github.com/atom/atom/pull/20524#issuecomment-821776315

**Reply by curtiscarlson** [reply]
Then they should roll it back until the other changes are made
